astroquery/ipac/irsa/most/core.py

```diff
@@ -0,0 +1,1017 @@
+"""
+MOST
```

Comment:
This narrative docs belongs to the rst file. We only auto generate the API docs at the bottom of it.
(moving this out should also fix the misparsing of most in the index page list)